src/components/ModalPortal.js

```diff
@@ -183,6 +184,13 @@ export default class ModalPortal extends Component {
         focusManager.popWithoutFocus();
       }
     }
+
+    if (
+      this.props.onAfterClose &&
+      typeof this.props.onAfterClose === "function"
```

Comment:
In case it is not a function, it's preferable to let it break and let the user fix it, otherwise we might hide a bug on the user's code.

Reply:
Sure, I'll update it then!